packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -617,12 +575,19 @@ export class GraphQLApi extends GraphqlApiBase {
    *
    * @experimental
    */
-  public appendToSchema(addition: string, delimiter?: string): void {
-    if ( this.schemaMode !== SchemaDefinition.CODE ) {
-      throw new Error('API cannot append to schema because schema definition mode is not configured as CODE.');
-    }
-    const sep = delimiter ?? '';
-    this.schema.definition = `${this.schema.definition}${sep}${addition}\n`;
+  public addToSchema(addition: string, delimiter?: string): void {
```

Comment:
NIT: I wonder if we need to surface this here, we could probably just allow users to call `api.schema.addToSchema` if they need the escape hatch.

Reply:
good idea.. might be misleading to other `schema-first` users as well...

Ill remove it and the `addObjectType` and `addInterfaceType` in the following PR :)

Reply:
actually nvm the queue to merge is fat.. ill remove it rn lmao